openedx/openedx-platform
# SE-1151 Campus.il Inlines all 36 discussion underscore templates

Performance issues were noted as scale with the discussion XBlock when theming was enabled, due to the massive number of underscore templates imported by the inline discussion template.

This change compiles all of these smaller files into a single underscore template file.

**JIRA tickets**: [OSPR-3621](https://openedx.atlassian.net/browse/OSPR-3621)

**Discussions**: Discussions on Campus Trello

**Sandbox URL**:

* LMS: https://pr20742.sandbox.opencraft.hosting/
* Studio: https://studio.pr20742.sandbox.opencraft.hosting/

**Merge deadline**: None

**Testing instructions**:

Automated tests should verify main functionality.

To test manually:

1. Create a course with an inline discussion block, e.g. [DemoX >  Lesson 3 - Be Social > Discussion Forums](https://pr20742.sandbox.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/courseware/social_integration/48ecb924d7fe4b66a230137626bfa93e/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%403c4b575924bf4b75a2f3542df5c354fc)
1. Ensure that all the buttons and functions available to the block still work.
1. Use the [main discussion forum](https://pr20742.sandbox.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/courseware/social_integration/48ecb924d7fe4b66a230137626bfa93e/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%403c4b575924bf4b75a2f3542df5c354fc), and ensure the same.

**Reviewers**
